apps/components/impact/latam-impact-section.tsx

```
@@ -0,0 +1,114 @@
+            <Text color="$white" textAlign="center" mt={8}>
+              Funds are released only when milestones are verified, ensuring every dollar drives measurable
+              impact.
+            </Text>
+          </ExplainerCard>
+        </MotiView>
+
+        {/* Impact Stats */}
+        <HStack space="md" justifyContent="space-between" flexWrap="wrap">
+          {impactStats.map((stat, index) => (
+            <MotiView
+              key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)